prometheus_flask_exporter/__init__.py

```diff
@@ -480,7 +480,9 @@ def _track(self, metric_type, metric_call, metric_kwargs, name, description, lab
             # merge the default labels and the specific ones for this metric
             combined = dict()
             combined.update(self._static_labels)
-            combined.update(labels)
```

Comment:
Could we do this instead?
```
if not labels:
  labels = self._static_labels
else:
  combined...
```

This makes it clear labels is sometimes not there and we don't need the inline comment.

Reply:
Yeah, np